public/index.html

```diff
@@ -19,8 +19,13 @@ <h1>TextSnippet</h1>
             <small></small>
         </div>
             <div class="button-container">
-                <button id="generate-snippet-btn">generate</button>
+                <button type="click" class="button" id="generate-snippet-btn">generate</button>
             </div>
+            <div class="muted">
+                <p id="output">speech muted</p>
```

Comment:
id should be more descriptive than "output" people who will read the code wont understand what is "output"